python/paddle/fluid/dygraph/dygraph_to_static/static_analysis.py

```diff
@@ -25,34 +25,41 @@
 
 # TODO: _is_paddle_dygraph_api is duplicated in Yamei's utils.py. Merge the two
 # function code together when Yamei finish her PR.
-def _is_paddle_dygraph_api(obj):
+def _is_api_in_module_helper(obj, module_prefix):
     m = inspect.getmodule(obj)
-    return m is not None and m.__name__.startswith("paddle.fluid.dygraph")
+    return m is not None and m.__name__.startswith(module_prefix)
 
 
 # TODO: is_dygraph_api is duplicated in Yamei's utils.py. Merge the two
 # function code together when Yamei finish her PR.
-def is_dygraph_api(node):
+def is_api_in_module(node, module_prefix):
     assert isinstance(node, gast.Call), "Input non-Call node for is_dygraph_api"
-    func_src = astor.to_source(node.func)
+    func_str = astor.to_source(node.func)
```

Comment:
Better to use `astor.to_source(gas.gast_to_ast(node.func))`.
Because `astor.to_source`  converts a ast.node but `node.func` is gast.node. Sometimes gast.node is different from ast.node and it will fail to convert a gast.node to source.

Reply:
Done